db/plans.go

```diff
@@ -0,0 +1,121 @@
+package db
+
+import (
+	"context"
+
+	t "github.com/cyverse-de/subscriptions/db/tables"
+	"github.com/doug-martin/goqu/v9"
+)
+
+func planDS(db GoquDatabase) *goqu.SelectDataset {
+	return db.From(t.Plans).
+		Select(
+			t.Plans.Col("id"),
+			t.Plans.Col("name"),
+			t.Plans.Col("description"),
+
+			t.PQD.Col("id").As(goqu.C("plan_quota_defaults.id")),
+			t.PQD.Col("plan_id").As(goqu.C("plan_quota_defaults.plan_id")),
+			t.PQD.Col("quota_value").As(goqu.C("plan_quota_defaults.quota_value")),
+			t.PQD.Col("resource_type_id").As(goqu.C("plan_quota_defaults.resource_type_id")),
+
+			t.RT.Col("id").As(goqu.C("resource_types.id")),
+			t.RT.Col("name").As(goqu.C("resource_types.name")),
+			t.RT.Col("unit").As(goqu.C("resource_types.unit")),
+		).
+		Join(t.PQD, goqu.On(t.PQD.Col("plan_id").Eq(t.Plans.Col("id")))).
+		Join(t.RT, goqu.On(t.PQD.Col("resource_type_id").Eq(t.RT.Col("id"))))
+}
```

Comment:
I haven't been able to get this to work. Every time I run queries similar to this, I get an error saying that the field corresponding to one of the columns can't be found. I don't think that Goqu supports automatically scanning structs that contain one-to-many relationships.